storage/database/dynamodb.go

```diff
@@ -406,6 +409,10 @@ func createBatchWriteWorker(writeCh <-chan *batchWriteWorkerInput) {
 		numUnprocessed := len(BatchWriteItemOutput.UnprocessedItems[batchInput.tableName])
 		for err != nil || numUnprocessed != 0 {
 			if err != nil {
+				if strings.Contains(err.Error(), "ValidationException") {
```

Comment:
Why does this error happen?
- If we remove the dynamoDB table, it can happen?

Reply:
It happens when there is a input errror such as duplicated input in batch, there is 0 or more than 25 items in batch, empty input in item and so on.
When we remove teh dynamoDB table, ResourceNotFoundException will occur. But this error can also occur when the table is being created or archived. I don't think it is a good idea to crit on ResourceNotFoundException.

Reply:
What is `duplicated input`? Is it same key/value item with another one?

Reply:
Yes, if there are two items with the same key, it is an error.

Reply:
OK. I got it your intention.
Please write some comments about the case of the error and why this crit is needed.